packages/terra-form-input/src/FormInput.scss

```diff
@@ -0,0 +1,103 @@
+:local {
+  .form-input {
+    background-color: var(--terra-form-input-background-color, #fff);
+    background-image: var(--terra-form-input-focus-animation-background, none);
+    border-color: var(--terra-form-input-border-color, #4a4a4a);
```

Comment:
Is this an issue here? https://github.com/cerner/terra-core/issues/1199

Reply:
Yeah, we'll need to merge the themable border styles into 1 shorthand border property for this @dylan-sftwr

Reply:
Done. 5e9eba487a5de8d996a8f96c29aa6381196df310